src/main/java/core/basesyntax/service/impl/ReportServiceImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.ReportService;
+import java.util.Map;
+
+public class ReportServiceImpl implements ReportService {
+    private static final String START_MESSAGE = "fruit,quantity";
+    private static final String REGEX = ",";
+    private StringBuilder builder = new StringBuilder()
+            .append(START_MESSAGE)
+            .append(System.lineSeparator());
```

Comment:
```suggestion
    private static final String COMMA = ",";
    private StringBuilder builder = new StringBuilder()
            .append(START_MESSAGE)
            .append(System.lineSeparator());
```
let's move builder inside `getReport()`